src/main/g8/build.sbt

```diff
@@ -16,7 +16,10 @@ lazy val root = (project in file("."))
       "org.http4s"      %% "http4s-dsl"          % Http4sVersion,
       "io.circe"        %% "circe-generic"       % CirceVersion,
       "org.specs2"      %% "specs2-core"         % Specs2Version % "test",
-      "ch.qos.logback"  %  "logback-classic"     % LogbackVersion
+      "ch.qos.logback"  %  "logback-classic"     % LogbackVersion,
+      $if(graal_native_image.truthy)$
+      "org.scalameta"   %% "svm-subs"            % "20.2.0"      % "compile-internal"
```

Comment:
I suspect this needs to align with the GraalVM version.  Is there any way to introspect it?